File: app/src/main/java/co/edu/uniautonoma/posgradosapp/Actividades/PqrsActivity.java
package co.edu.uniautonoma.posgradosapp.Actividades;

import android.content.Intent;

import androidx.appcompat.app.AppCompatActivity;
import androidx.lifecycle.ViewModelProviders;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import co.edu.uniautonoma.posgradosapp.R;
import co.edu.uniautonoma.posgradosapp.ViewModels.PqrsViewModel;
import xdroid.toaster.Toaster;

public class PqrsActivity extends AppCompatActivity {

    private EditText etComentario;
    private String destinatario;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pqrs);

        etComentario = (EditText) findViewById(R.id.etComentario);
        Button btEnviar = (Button) findViewById(R.id.btEnviar);

        PqrsViewModel viewModel = ViewModelProviders.of(this).get(PqrsViewModel.class);
        if (viewModel.getEscuela()!=null){
            viewModel.getEscuela().observe(this, escuela -> {
                destinatario = escuela.getCorreo();
            });
        }else {
            Toaster.toast(R.string.EstadoServidor);
        }

        btEnviar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent emailIntent = new Intent(Intent.ACTION_SEND);

                String comentario = etComentario.getText().toString();

                emailIntent.putExtra(Intent.EXTRA_EMAIL, new String[]{destinatario});
                emailIntent.putExtra(Intent.EXTRA_SUBJECT, getResources().getString(R.string.asuntopqrs));
                emailIntent.putExtra(Intent.EXTRA_TEXT, comentario);
                emailIntent.setType("message/rfc822");

                startActivity(Intent.createChooser(emailIntent,"Email"));
            }
        });
    }
}

File: presentation/src/main/java/co/edu/uniautonoma/posgradosapp/presentation/ui/pqrs/PqrsActivity.kt
package co.edu.uniautonoma.posgradosapp.presentation.ui.pqrs

import android.content.Intent
import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import androidx.lifecycle.Observer
import co.edu.uniautonoma.posgradosapp.presentation.R
import co.edu.uniautonoma.posgradosapp.presentation.ui.base.BaseActivity
import kotlinx.android.synthetic.main.activity_pqrs.*
import org.koin.androidx.viewmodel.ext.android.viewModel
import xdroid.toaster.Toaster

class PqrsActivity : BaseActivity() {

    private val pqrsViewModel: PqrsViewModel by viewModel()
    private var destinatario = ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pqrs)

        observarResultados()
        enviarCorreo()
    }

    private fun observarResultados() {
        pqrsViewModel.destinatarioLiveData.observe(this, Observer {
            destinatario = it
        })

        pqrsViewModel.errorLiveData.observe(this, Observer {
            Toaster.toast(R.string.EstadoServidor)
            Log.d("PqrsError:", it)
        })
    }

    private fun enviarCorreo() {
        btEnviar.setOnClickListener {
            val emailIntent = Intent(Intent.ACTION_SEND)
            val comentario = etComentario!!.text.toString()

            emailIntent.putExtra(Intent.EXTRA_EMAIL, arrayOf(destinatario))
            emailIntent.putExtra(Intent.EXTRA_SUBJECT, resources.getString(R.string.asuntopqrs))
            emailIntent.putExtra(Intent.EXTRA_TEXT, comentario)
            emailIntent.type = "message/rfc822"
            startActivity(Intent.createChooser(emailIntent, "Email"))
        }
    }
}